apiserver/v1/types_apirequestcount.go

```diff
@@ -62,14 +62,14 @@ type APIRequestCountStatus struct {
 	// requestsInTheCurrentHour contains request history for the current hour. This is porcelain to make the API
 	// easier to read by humans seeing if they addressed a problem. This field is reset on the hour.
 	// +optional
-	RequestsLastHour PerResourceAPIRequestLog `json:"requestsInTheCurrentHour"`
+	RequestsInTheCurrentHour PerResourceAPIRequestLog `json:"requestsInTheCurrentHour"`
 
 	// requestsInTheLast24h contains request history for the last 24 hours, indexed by the hour, so
 	// 12:00AM-12:59 is in index 0, 6am-6:59am is index 6, etc. The index of the current hour
 	// is updated live and then duplicated into the requestsLastHour field.
 	// +kubebuilder:validation:MaxItems=24
 	// +optional
-	RequestsLast24h []PerResourceAPIRequestLog `json:"requestsInTheLast24h"`
+	RequestsInTheLast24h []PerResourceAPIRequestLog `json:"requestsInTheLast24h"`
```

Comment:
just `last24h`?

Reply:
> just `last24h`?

I don't personally like it, but I don't care enough to argue.  You sure you want to eliminate these?  It makes the code difficult for me to read in usage.